doc/data_mover_threads_new.3.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(DATA_MOVER_THREADS_NEW, 3)
+collection: miniasync
+header: DATA_MOVER_THREADS_NEW
+secondary_title: miniasync
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2022, Intel Corporation)
+
+[comment]: <> (data_mover_threads_new.3 -- man page for miniasync data_mover_threads_new operation)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**data_mover_threads_new**(), **data_mover_threads_delete**(),
+**data_mover_threads_default**() - allocate, free or allocate with default parameters
```

Comment:
The double `allocate` makes it a bit confusing to me. What's the difference between allocate and allocate with default parameters in this case?

Reply:
Okay, never mind, I just got confused. I see that this describes multiple functions now.